classes/request/post/class-kom-request-post-refund.php

```diff
@@ -106,7 +106,7 @@ public function get_refund_order_lines() {
 						if ( $item->get_product_id() === $order_item->get_product_id() ) {
 							$order_line_total    = round( ( $order->get_line_subtotal( $order_item, false ) * 100 ) );
 							$order_line_tax      = round( ( $order->get_line_tax( $order_item ) * 100 ) );
-							$order_line_tax_rate = ( 0 != $order_line_tax && 0 != $order_line_total ) ? round( ( $order_line_tax / $order_line_total ) * 100 * 100 ) : 0; // phpcs:ignore WordPress.PHP.StrictComparisons -- Non strict is ok here.
+							$order_line_tax_rate = empty( $order_line_tax + $order_line_total ) ? 0 : reset( WC_Tax::get_base_tax_rates( $product->get_tax_class() ) )['rate'] * 100 ?? round( ( $order_line_tax / $order_line_total ) * 100 * 100 );
```

Comment:
I've made the necessary adjustments. 

Regarding your last paragraph, it should not throw an error since if we check if `['rate']` exists on an empty array, the null coalescing operator will be triggered rather than an error being thrown.